daos-stack/daos
# DAOS-10912 test: Verify EC space consumption

Test-tag: pr EcodSpaceUsage HarnessUnitTest storage_tiers PoolCreateAllHwTests VmdLedStatus,-manual
Skip-unit-tests: true
Skip-fault-injection-test: true

- Added ftest/erasurecode/space_usage
  - Verify space consumption of EC object classes is balanced within 5%
- Added dmg_utils.py:pool_query_targets()
- Added TestPool.query_targets()
- Added TestPool.get_space_per_target()
- Added ftest/utils/data_utils.py
  - Misc utilities related to data manipulation
- Remove yaml_utils.py:find_values in favor of data_utils.py:dict_extract_values
- Added ftest/harness/unit
  - Harness unit tests for utilities
  - 
Required-githooks: true

### Before requesting gatekeeper:


### Gatekeeper:
